test_util/cluster.py

```diff
@@ -322,16 +324,12 @@ def install_dcos(
 
 def upgrade_dcos(cluster, installer_url, add_config_path=None):
 
-    def upgrade_host(tunnel, role, bootstrap_url):
-        # Download the install script for the new DC/OS.
-        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/dcos_install.sh'])
+    def upgrade_host(tunnel, role, bootstrap_url, upgrade_script_path):
+        # Download the upgrade script for the new DC/OS.
+        tunnel.remote_cmd(curl_cmd + ['--remote-name', upgrade_script_path + 'dcos_node_upgrade.sh'])
```

Comment:
This would break if we ever renamed the upgrade script. This function should take the complete URL that was printed to stdout.